src/utl/CMakeLists.txt

```diff
@@ -69,15 +71,37 @@ target_sources(utl
     src/LoggerCommon.cpp
     src/MakeLogger.cpp
     src/Metrics.cpp
+    src/CFileUtils.cpp
+    src/ScopedTemporaryFile.cpp
 )
   
 target_include_directories(utl
   PUBLIC
     include
     src
+    ${Boost_INCLUDE_DIRS}
```

Comment:
You should put boost in target_link_libraries

Reply:
Since span is header-only I don't think that's necessary, but LMK if there's a reason you're thinking we'd want it anyway.